resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnShuffleIntegrationSuite.scala

```diff
@@ -109,6 +110,59 @@ class YarnShuffleAuthSuite extends YarnShuffleIntegrationSuite {
 
 }
 
+/**
+ * SPARK-34828: Integration test for the external shuffle service with an alternate name and
+ * configs (by using a configuration overlay)
+ */
+@ExtendedYarnTest
+class YarnShuffleAlternateNameConfigSuite extends YarnShuffleIntegrationSuite {
```

Comment:
Please make this new test suite as a separate file.

Reply:
Sure. I was following the example of `YarnShuffleAuthSuite`. Moved out to a separate file.